RecoTauTag/RecoTau/plugins/PFRecoTauChargedHadronFromTrackPlugin.cc

```diff
@@ -143,13 +143,14 @@ PFRecoTauChargedHadronFromTrackPlugin::return_type PFRecoTauChargedHadronFromTra
   evt.getByToken(Tracks_token, tracks);
 
   qcuts_->setPV(vertexAssociator_.associatedVertex(jet));
-
+  float jEta=jet.eta();
+  float jPhi=jet.phi();
   size_t numTracks = tracks->size();
   for ( size_t iTrack = 0; iTrack < numTracks; ++iTrack ) {
     reco::TrackRef track(tracks, iTrack);
 
     // consider tracks in vicinity of tau-jet candidate only
-    double dR = deltaR(track->eta(), track->phi(), jet.eta(), jet.phi());
+    double dR = deltaR(track->eta(), track->phi(), jEta,jPhi);
```

Comment:
isn't taking out an obvious const x.func() call out of the loop just a syntactic improvement at this point?
I played a bit in the past with some simple compile units with simple functions and then looking at assembly instructions. Even O1 already takes out repeated function calls outside of the loop.